website/templates/share_search.mako

```diff
@@ -6,4 +6,16 @@
 
 <%def name="javascript_bottom()">
     <script src=${"/static/public/js/share-search-page.js" | webpack_asset}></script>
+
+<script type="text/x-mathjax-config">
+   MathJax.Hub.Config({
+       tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']], processEscapes: true}
```

Comment:
Okay. I only find 12 out of 1838 (0.7%) arXiv docs with `emph{` in the description, so I guess it's not worth worrying about.

```
find archive/arxiv_oai -name raw.json -exec grep emph\{ {} \;|wc -l
```